tests/qadence/test_noise/test_digital_noise.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import random
+
+import pytest
+import strategies as st  # type: ignore
+import torch
+from hypothesis import given, settings
+
+from qadence import (
+    DigitalNoiseType,
+    H,
+    NoiseHandler,
+    QuantumCircuit,
+    QuantumModel,
+    Z,
+    hamiltonian_factory,
+    kron,
+    set_noise,
+)
+
+list_noises = [DigitalNoiseType(noise.value) for noise in DigitalNoiseType]
```

Comment:
Since it is reused in more than one test, maybe it could become a fixture ?

Reply:
So far, better to keep the parameterize on the list instead of doing a for loop in the test.